react/README.md

```diff
@@ -21,3 +28,10 @@
 - [Deploy Create React App project to S3](./docs/cra-s3-compressed-deploy.md)
 
 - [Testing guide with Jest & Enzyme](./docs/jest-enzyme-testing.md)
+
+## Posts
+- [Introducing redux-recompose](https://medium.com/wolox-driving-innovation/932e746b0198)
+
+  #### React Native
+  - [Crear un módulo nativo con React Native](https://medium.com/wolox-driving-innovation/crear-un-m%C3%B3dulo-nativo-con-react-native-5d71a85c7dc6)
+  - [Servicios de Android mediante Módulos Nativos de React Native](https://medium.com/wolox-driving-innovation/servicios-de-android-mediante-m%C3%B3dulos-nativos-de-react-native-d78e883522b4)
```

Comment:
Also:
https://medium.com/wolox-driving-innovation/https-medium-com-wolox-driving-innovation-easy-forms-in-react-native-with-redux-form-1cdc16a9a889

Reply:
https://medium.com/wolox-driving-innovation/react-native-heatmap-cc29181e6276

Reply:
neither these 😲